crates/backend/src/codegen.rs

```diff
@@ -523,8 +523,12 @@ impl TryToTokens for ast::Export {
             argtys.push(&*arg.ty);
             let i = i + offset;
             let ident = Ident::new(&format!("arg{}", i), Span::call_site());
-            let ty = &arg.ty;
-            match &*arg.ty {
+            let ty = match &*arg.ty {
+                syn::Type::Group(syn::TypeGroup { ref elem, .. }) => elem,
+                _ => &arg.ty,
+            };
```

Comment:
Could you also handle `Type::Paren` while you're at it?

Also, it might be a good idea to use a `while let` here in case there are multiple nested `Type::Paren`s / `Type::Group`s.